rubrik_cdm/rubrik_cdm.py

```diff
@@ -112,6 +112,19 @@ def __init__(self, node_ip=None, username=None, password=None, api_token=None, e
                     self.password = password
                     self.log("Password: *******\n")
 
+                # Domain display name may be specified to force authentication to this domain
+                session_url = 'https://{}/api/internal/session'.format(self.node_ip)
+                ddn = domain_display_name if domain_display_name else os.environ.get('rubrik_cdm_domain')
+                if ddn:
+                    session_url = session_url + '/realm/{}'.format(ddn)
+
+                # Get the web API session token to make future interactions faster
+                response = requests.post(session_url, data='{"initParams": {}}', auth=(self.username, self.password), verify=False)
```

Comment:
What happens if this call fails? Connectivity issues etc.

We also need to have a timeout value of some sort here. Default throughout the SDK is 15 seconds but it's not unheard-of for customers to need to increase that.